content/en/blog/2023/jaeger-exporter-collector-migration.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Migrating away from the Jaeger exporter in the Collector
+linkTitle: Migrating away from the Jaeger exporter in the Collector
+date: 2023-09-14
+author: >-
+  [Alex Boten](https://twitter.com/codeboten) (Lightstep)
+cSpell:ignore:
+---
+
+The
+[latest binary](https://github.com/open-telemetry/opentelemetry-collector-releases/releases/tag/v0.85.0)
+release of the OpenTelemetry Collector no longer includes exporters for the
+native Jaeger format. This change was prompted because Jaeger has support for
+OTLP out of the box. If you are a current user of either the Jaeger or Jaeger
+Thrift exporter, you have a couple of options to choose from when upgrading to
+the latest Collector.
+
+## Switch to OTLP exporter
+
+The recommended option is to switch to the OpenTelemetry Protocol exporter. This
+exporter is included in all the official distributions supported by the
+community. To switch, you'll need to update the collector configuration. Find
+the following Jaeger exporter configuration:
+
+```yaml
+exporters:
+  jaeger:
+    endpoint: https://jaeger.example.com:14250
+
+service:
+  pipelines:
+    exporters: [jaeger]
+```
+
+And replace it with the OTLP configuration:
+
+```yaml
+exporters:
+  otlp/jaeger: # Jaeger supports OTLP directly
+    endpoint: https://jaeger.example.com:4317
+
+service:
+  pipelines:
+    exporters: [otlp/jaeger]
+```
+
+Depending on your version of Jaeger, you may need to set the following
+environment variable `COLLECTOR_OTLP_ENABLED=true`.
```

Comment:
can you put a bound here like "since version X this is not required anymore"

Reply:
This is a good question, maybe @pavolloffay or @yurishkuro can confirm what the right versions are here. I onlny tested this w/ the all-in-one container versions 1.35 & 1.41.

Reply:
from v1.46.0 there is no need to set COLLECTOR_OTLP_ENABLED=true